### DISCUSSION TITLE
blog/20130206/

### DISCUSSION THREAD

**Discussion by giscus-x** [xaoxuu/blog-comments#49] [Announcements]
# blog/20130206/

本文只是我在大学时期关于小组决策效率与分工方面的思考，可能不适用于其它场景。 在任何一个团队中，我始终都是先做一个观察者，因为在掌握足够的信息之前，我的看法可能是片面的，由此产生的观点及推论发表出来可能会误导人。

https://xaoxuu.com/blog/20130206/

**Comment by fenychn0206** [comment]
不错的文章，那个时候还没🔥吗？

**Reply by xaoxuu** [reply]
那时候忘了是发在Qzone还是哪的，17年才有的这个网站。
